### PR TITLE
Feature/two letter iso codes

### LINKED ISSUE #822: Document --serverinfo two-letter ISO country code support
https://github.com/jamulussoftware/jamulus/pull/2841 should be documented on the website.
We should make the ISO-code-based variant the preferred one and only keep the documentation for "Jamulus 3.9.0 or earlier".

Note: This is nice-to-have for 3.9.1, but in no way mandatory as it doesn't break existing documentation in any way.

### PR DESCRIPTION
**Pull request by pljones**
**Short description of changes**
Update `--serverinfo` to mention two letter country codes.

**Context: Fixes an issue? Related issues**
FIXES: #822 
Relates to https://github.com/jamulussoftware/jamulus/pull/2841

**Status of this Pull Request**
Needs review.  Contains external link.

**What is missing until this pull request can be merged?**
Built and test - looks okay to me:
http://jamuluswebsite.drealm.info/wiki/Running-a-Server

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch (I typed "YES" to translations and PR goes to next-release and I've checked that ... yes I have.  Definitely.)

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
Not sure if what removing the whitespace implies. But all in all, I think it's good.

@gilgongo @ignotus666 do you know anything about the whitespace?

**Comment by pljones** [comment]
> Not sure if what removing the whitespace implies. But all in all, I think it's good.
> 
> @gilgongo @ignotus666 do you know anything about the whitespace?

I checked before and after builds:

http://jamuluswebsite.drealm.info/wiki/Software-Manual#command-line-options
https://jamulus.io/wiki/Software-Manual#command-line-options

The whitespace isn't meaningful.  Ideally, for the website, it would be a table

| short | long | description |
|---|---|---|
| -a | --a-long-option | what it does |
| | --no-short-option | what this one does |

but that was a bit of a bigger change than I felt was appropriate here..! :)

File: wiki/en/Include-Server-Commands.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-- `-P`  or `--delaypan`       Start with delay panning enabled See [notes](#delay-panning)
-- `-R`  or `--recording`      Set server recording directory; Server will record when a session is active by default. See [Options](#options).  
-- `--norecord`                Set server not to record by default (when recording is configured e.g via `-R`)
+- `-f` or `--listfilter` Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only.
+- `-F` or `--fastupdate` Reduces latency if Clients connect with "Enable Small Network Buffers" option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
+- `-l` or `--log` Enable logging, set path and file name
+- `-L` or `--licence` Show an agreement window before users can connect
+- `-m` or `--htmlstatus` Enable HTML status file, set path and file name
+- `-o` or `--serverinfo` Location details in the format: `[name];[city];[country as two-letter ISO country code or Qt5 Locale]` (see [two-letter ISO country codes](http://statoids.com/wab.html) ("A-2" column) or [Qt5 Locale values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only
```
Comment (hoffie) [inline wiki/en/Include-Server-Commands.md]: Thanks!

Could we also use Wikipedia as a more neutral and maybe persistent source here?

```suggestion
- `-o` or `--serverinfo` Location details in the format: `[name];[city];[country as two-letter ISO country code or Qt5 Locale]` (see [two-letter ISO country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Current_codes) or [Qt5 Locale values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only
```

Comment (pljones) [reply wiki/en/Include-Server-Commands.md]: You mean https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements ?  Yes, that'd be better.

For some reason, I couldn't find the Wikipedia page.  (I think Google didn't offer it to my search - "ISO_3166-1" was probably too vague a match :) ).

Comment (hoffie) [reply wiki/en/Include-Server-Commands.md]: > You mean https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements ? Yes, that'd be better.

Yeah, that section is better than my suggestion. :)

**Review by hoffie — APPROVED** [review]
Thanks!

(Can't comment on the relevance of the whitespace changes)

**Pull request merged**